frontend/src/router.ts

```diff
@@ -46,7 +46,7 @@ export const router = new Router({
             activator: true,
             activatorName: "Sound Quality Reporting",
             menuIcon: "fas fa-headphones",
-            auth: { requireClaims: { coordinator: true } }
+            auth: { requireClaims: ["SQR.coordinator"] }
```

Comment:
Let's replace `requireCLaims` router metadata key with `roles` key for specifying roles for which the route is accessible. Also we can support both `role: string` and `roles: string[]` for convenience.